downstream/modules/platform/proc-specify-nodes-job-execution.adoc

```diff
@@ -0,0 +1,88 @@
+[id="proc-specify-nodes-job-execution"]
+
+= Specify nodes for job execution
+
+A node selector can be added to the container group pod specification to ensure they only run against certain nodes.  
+First add a label to the nodes you want to run jobs against. 
+
+The following procedure adds a label to a node.
+
+.Procedure
+. List the nodes in your cluster, along with their labels:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+kubectl get nodes --show-labels
+----
++
+The output is similar to this:
++
+[cols="15%,15%,15%,15%,15,25%",options="header"]
+|====
+| Name | Status | Roles | Age | Version | Labels
+| `worker0` | Ready | <none> | 1d | v1.13.0 | `...,kubernetes.io/hostname=worker0`
+| `worker1` | Ready | <none> | 1d | v1.13.0 | `...,kubernetes.io/hostname=worker1`
+| `worker2` | Ready | <none> | 1d | v1.13.0 | `...,kubernetes.io/hostname=worker2`
+|====
++
+. Choose one of your nodes, and add a label to it:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+kubectl label nodes <your-node-name> <aap_node_type>=<execution>
+----
++
+For example:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+kubectl label nodes <your-node-name> disktype=ssd
+----
++
+where `<your-node-name>` is the name of your chosen node.
++
+. Verify that your chosen node has a `disktype=ssd` label:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+kubectl get nodes --show-labels
+----
++
+. The output is similar to this:
+
++
+[cols="15%,15%,15%,15%,15,25%",options="header"]
+|====
+| Name | Status | Roles | Age | Version | Labels
+| `worker0` | Ready | <none> | 1d | v1.13.0 | `...disktype=ssd,kubernetes.io/hostname=worker0`
+| `worker1` | Ready | <none> | 1d | v1.13.0 | `...,kubernetes.io/hostname=worker1`
+| `worker2` | Ready | <none> | 1d | v1.13.0 | `...,kubernetes.io/hostname=worker2`
+|====
++
+In the preceding output, you can see that the `worker0` node has a `disktype=ssd` label.
++
+. In the {ControllerName} UI, specify that label in your pod specification on the container group.  
```

Comment:
Where is that label  specified here?

Reply:
Labels are in the metadata section of the custom pod spec. See
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/